src/common/user-store/preferences-helpers.ts

```diff
@@ -22,9 +22,11 @@
 import moment from "moment-timezone";
 import path from "path";
 import os from "os";
-import { ThemeStore } from "../../renderer/theme.store";
+import type { ThemeId } from "../../renderer/theme.store";
 import { ObservableToggleSet } from "../utils";
 
+export const defaultThemeName: ThemeId = "lens-dark";
```

Comment:
is this to avoid a circular dep? Seems unfortunate not be able to use the already defined default value